core-services/prow/02_config/openshift/ansible-operator-plugins/_prowconfig.yaml

```diff
@@ -73,7 +73,7 @@ tide:
     - approved
     - jira/valid-reference
     - lgtm
-    - qe-approved,no-qe
+    - qe-approved
```

Comment:
Thanks for removing the `no-qe` label from the prowconfig. It cannot be applied as it not available in the openshift repos. Ref: https://github.com/openshift/ansible-operator-plugins/pull/7#issuecomment-2276243875